### DISCUSSION TITLE
What kind of fruit is greengage? 🍈

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1103] [Galaxy Trivia] [answered]
**Bonus:** In which 🇺🇸 state is it illegal to cook 🥬?

**Comment by peterbaccaro** [comment]
A greengage is a type of fruit that belongs to the **plum** family (Prunus domestica). It is a cultivar of plum known for its greenish colour and sweet, juicy flesh. Greengages are often considered a type of "true plum" and are known for their distinctive flavour, which can range from tart to very sweet, depending on the variety and ripeness.

In **West Virginia** its illegal to cook cabbage because of smell.

**Comment by NomanGhiasy** [comment]
The greengages are a group of cultivars of the common European plum. Gages are small, round and sweet, but grow best in a sunny spot.

**Comment by thuongtruong1009** [comment, marked as answer]
A greengage is a type of fruit that belongs to the plum family. It is known for its sweet and slightly tart flavor. Greengages are small, round, and typically green or yellow when ripe.

In the United States, it is illegal to cook collard greens (🥬) in the state of North Carolina on a shared rental property, such as an apartment building. This unusual law is intended to prevent strong odors from cooking collard greens and similar vegetables from disturbing neighbors. However, it's important to note that local laws and regulations can change, so it's always a good idea to check the current regulations if you have specific concerns.

**Comment by sheri-na** [comment]
Greengage is a European plum and it is illegal to cook cabbage 🥬 in this case but also sauerkraut in West Virginia due to the odors

**Comment by Mayathebeeee** [comment]
I dont know !
